app/components/molecules/promotion-card/promotion-card.component.tsx

```diff
@@ -0,0 +1,28 @@
+import {FC} from 'react';
+
+import {
+  BadgeContainer,
+  Img,
+  PromotionCardFooter,
+  PromotionCardProps,
+  StyledPromotionCard,
+} from './promotion-card.styles';
+import {Badge} from '../../atoms/badge';
+
+export const PromotionCard: FC<PromotionCardProps> = ({
```

Comment:
you can use `(props) =>` so you dont have to write them twice, _i.e: you create the `interface` and then you extract everything in here!_ 

and use it in the component like:

```typescript
import {FC} from 'react';

import {
  BadgeContainer,
  Img,
  PromotionCardFooter,
  PromotionCardProps,
  StyledPromotionCard,
} from './promotion-card.styles';
import {Badge} from '../../atoms/badge';

export const PromotionCard: FC<PromotionCardProps> = ((props) => {
  const {
    variant = 'default',
    bgImage,
    label,
  } = props;

  return (
    <StyledPromotionCard>
      {variant !== 'default' && (
        <BadgeContainer>
          <Badge variant={variant} />
        </BadgeContainer>
      )}
      <PromotionCardFooter>{label}</PromotionCardFooter>
      <Img src={bgImage} />
    </StyledPromotionCard>
  )
});
```

And now we see `bgImage`, I think thats a very bad name for the `image` haha.

What about `promoImageUrl` so it's a string, we know that it's gonna be a URL.

So that way you could use it like this:

```typescript
<PromotionCard 
    variant={args.variant}
    promoImageUrl={args.promoImageUrl}
    label={args.label}
  />
```